configCivcraft.yml

```diff
@@ -2836,11 +2836,11 @@ recipes:
     input:
       diamond ore:
         material: DIAMOND_ORE
-        amount: 20
+        amount: 32
     output:
       diamond:
         material: DIAMOND
-        amount: 24
+        amount: 64
```

Comment:
I wouldnt make the ratio >1.33 for diamonds, because otherwise we get the same situation as in 2.0, where anyone without a lategame ore smelter can't compete with a player with one at all, because he would have to mine twice the time or something like that. The smelter should be a decent investment that takes a while to pay off and not neccessity for anyone who ever wants to mine diamonds.